tsfc/parameters.py

```diff
@@ -14,8 +14,11 @@
     # that makes compilation time much shorter.
     "unroll_indexsum": 3,
 
-    # Scalar type (C typename string)
-    "scalar_type": "double",
+    # Scalar type numpy dtype
+    "scalar_type": numpy.dtype(numpy.float64),
+
+    # So that tests pass (needs to match scalar_type)
+    "scalar_type_c": "double",
```

Comment:
This is a really dumb way to do this (scalar_type_c can be expunged when coffee backend goes).